OTAnalytics/domain/flow.py

```diff
@@ -27,31 +36,147 @@ class Flow:
         ValueError: if distance is negative
     """
 
+    id: FlowId
+    name: str
+    start: SectionId
+    end: SectionId
+    distance: float
+
     def __init__(
```

Comment:
Your comment is right for dataclasses. The dataclass decorator is a relict of an early definition of the flow. I removed it because the `Flow` is no longer a dataclass.